app/helpers/activity_helper.rb

```diff
@@ -47,4 +47,13 @@ def benefitting_countries_with_percentages(benefitting_countries)
       )
     end
   end
+
+  def edit_comment_path_for(commentable, comment)
+    case commentable.class.name
+      when "Activity"
+        edit_activity_comment_path(commentable, comment)
+      when "Actual"
```

Comment:
This feels like it's missing other commentables that might be relevant (e.g. Refunds). Activity is special (as something that is it's own activity) so I think this should be Activity vs. Everything Else.

Reply:
They can't edit refund and adjustment comments from the activity view right now, so we thought it would be premature featurisation.

Reply:
For example, the decision to allow editing comments on actuals by editing the entire actual was just that, a decision, which I wouldn't want to impose on all other similarly shaped objects. Edit paths for comments on refunds might end up linking to the comment rather than the actual, and I don't want to make that decision now.

Reply:
:+1: